src/types/twitter-downloader.model.ts

```diff
@@ -1,16 +1,65 @@
-export default interface TwitterDownloaderResponse{
-    result: {
-        tweet_meta_data: {
-            tweet_url: string,
-            uploader_user_id: string,
-            uploader_display_name: string,
-            upload_date: string,
-            duration: string,
-            thumbnail: string,
-            title: string,
-            description: string
-        },
-        tweet_medias: object,
-        error: string
-    }
-}
\ No newline at end of file
+/** Interface for TweetMeta */
+export interface TweetMeta {
+    /** URL of the tweet */
+    tweet_url: string;
+
+    /** ID of the user who uploaded the tweet */
+    uploader_user_id: string;
+
+    /** Display name of the user who uploaded the tweet */
+    uploader_display_name: string;
+
+    /** Date the tweet was uploaded */
+    upload_date: string;
+
+    /** Duration of the tweet media */
+    duration: string;
+
+    /** URL of the thumbnail for the tweet media */
+    thumbnail: string;
+
+    /** Title of the tweet */
+    title: string;
+
+    /** Description of the tweet */
+    description: string;
+}
+
```

Comment:
Inconsistent semicolon usage between this file and `src/types/tiktok-downloader.model.ts`. Both are valid, but we must be consistent